modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+* You have the openshift-installer binary
+
+.Procedure: 
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshfit-install 4.15.z
+built from commit e80d50ac8c9ab65833045d648a01eb1a0a3a6d7d
```

Comment:
@jeffdyoung So since this is an example output, I don't think its necessary to put a open ended value here. Usually those updatable values (usually formated like this <image>) are for commands, so that the user can update them. 
I did update it on [line 40 ](https://github.com/openshift/openshift-docs/pull/69883/files#diff-01f0b2e4a4420fd86c18783d418edd6deb180ba575271bfd5a21cec9469f3f43R40) so its still like a value.